### PR TITLE
Update projects.js

### PR DESCRIPTION
**Pull request by meetmangukiya**
Not sure if this is big enough or quality wise qualifiable as a GSoC project

### CODE REVIEW

**Comment by sims1253** [comment]
md is missing

**Comment by jayvdb** [comment]
Can you use a commit shortlog like "projects.js: Add coala statistics"

**Comment by meetmangukiya** [comment]
@jayvdb done, but does this look like a potential GSoC project ?

File: data/projects/coala_stats.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Statistics like :
+
+#### Normal stats
+- How many times was coala ran?
```
Comment (sims1253) [inline data/projects/coala_stats.md]: was run

**Comment by sims1253** [comment]
So your idea is to have coala report to us?
like the "send anonymous user stats" thing everyone does?

I personally don't like this but that is an ideological concern.

**Comment by meetmangukiya** [comment]
Also allow users to see for themselves, like github statistics @sims1253

**Comment by Udayan12167** [comment]
Just one concern over here over the accepted patches statistic. A lot of the times I let coala suggest changes and then manually go and change them without accepting time. A lot of the issues might be communicated to coala through ci. These factors will skew the statistics a lot. Other than that just showing number of suggested patches by coala could also be bad because of the number of people trying coala as newcomers. We need to segregate on the basis of repository basis or size of codebase that coala was run on. IMO it needs better definition the project.

File: data/projects/coala_stats.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+#### Misc. 
+- Repository specific statistics, Bear specific statistics and global statistics.
+- How many changes did coala bring to the source code? 
+
+#### Using the data for promotion
+- Option to permit coala transmit these data to the coala server. 
+- Showcase the visualisations of the data on the coala website. 
+
+Data visualisations are a good means of promotions and also to prove the power of coala(and help it conquer the world eventually.)
```
Comment (Mariatta) [inline data/projects/coala_stats.md]: nitpick, line seems too long.
Do we follow certain style guide for this?
If not, ignore.

Comment (Mariatta) [inline data/projects/coala_stats.md]: also space before ``(and help)``

**Comment by sims1253** [comment]
I don't feel like this project has lots of supporters so I am closing the pr. Feel free to reopen if you disagree

**Comment by meetmangukiya** [comment]
I think someone on gsoc channel is working again on this PR, however that'd
raise a new PR :+1:

**Pull request closed without merging**